.evergreen/config/functions.yml

```diff
@@ -167,9 +167,19 @@ functions:
   "bootstrap mongo-orchestration":
     - command: shell.exec
       params:
+        include_expansions_in_env:
```

Comment:
Noted that `vars` (when calling a `func`) actually define expansions (per [Functions](https://docs.devprod.prod.corp.mongodb.com/evergreen/Project-Configuration/Project-Configuration-Files#functions)). That distinction wasn't clear to me before this PR.